test/test_temporal_interpolate.py

```diff
@@ -68,7 +69,7 @@ def test_cube_timemean():
     S, N, W, E = 34, 35, -117, -116
     date       = 20200130
     dct_hrs    = {'GMAO': [12, 15, '13:30:00'], 'ERA5': [13, 14, '13:30:00']}
-    hr1, hr2, ti = dct_hrs[WM]
+    hr1, hr2, ti = dct_hrs[wm]
```

Comment:
i noticed the changing from WM and wm back and forth. Rationale?

Reply:
Some files are called 'ERA-5' and others are 'ERA5'. I probably have some wrong somewhere still; will update

Reply:
Yes, this seems like a code hack to make it work rather than going to the root of the cause.

Reply:
it's just the weather model file name